packages/shared/src/components/post/SharePostContent.tsx

```diff
@@ -84,7 +92,9 @@ function SharePostContent({
             </div>
             <div className="block overflow-hidden ml-2 w-70 rounded-2xl cursor-pointer h-fit">
               <LazyImage
-                imgSrc={post.sharedPost.image}
+                imgSrc={
+                  post.sharedPost.image || cloudinary.post.imageCoverPlaceholder
```

Comment:
We should probably put the value at `fallbackImage` so it will have wider coverage (deleted images, etc.)

Reply:
Moved to LazyImage here f314ef3cb303c557f7d1f47ba2b223413fda848d